File: interview_preparation/capg/Coding/standard_role/Easy/count_vowels_in_a_string.cpp
//Write a function that counts the number of vowels (`a`, `e`, `i`, `o`, `u`) in a given string.

#include <iostream>
#include <string>

using namespace std;

int countVowels(string str) {
    int count = 0;
    for (char ch : str) {
        char lowerCh = tolower(ch); // Make case-insensitive
        if (lowerCh == 'a' || lowerCh == 'e' || lowerCh == 'i' || lowerCh == 'o' || lowerCh == 'u') {
            count++;
        }
    }
    return count;
}

int main() {
    string str = "Hello, World!";
    int vowelCount = countVowels(str);
    cout << "Number of vowels: " << vowelCount << endl;
    return 0;
}

File: interview_preparation/capg/Coding/standard_role/Easy/count_vowels_in_a_string.py
#Write a function that counts the number of vowels (`a`, `e`, `i`, `o`, `u`) in a given string.

def count_vowels(string):
    count = 0
    for char in string:
        if char in "aeiou":
            count += 1
    return count

print(count_vowels("You Gange"))